packages/retail-ui/components/ComboBox/__tests__/ComboBox-test.tsx

```diff
@@ -1155,4 +1155,57 @@ describe('ComboBox', () => {
       });
     });
   });
+
+  describe('Locale', () => {
+    let search: jest.Mock<Promise<any>>;
+    let promise: Promise<any>;
+    let wrapper: ReactWrapper;
+    beforeEach(() => {
+      [search, promise] = searchFactory(Promise.resolve(null));
+    });
+    const focus = async (): Promise<void> => {
+      (wrapper.find(ComboBox).instance() as ComboBox<any>).focus();
+      await promise;
+      wrapper.update();
+    };
+
+    it('render default locale', async () => {
+      wrapper = mount(<LocaleProvider><ComboBox getItems={search} /></LocaleProvider>);
```

Comment:
```suggestion
      wrapper = mount(<ComboBox getItems={search} />);
```
Здесь или дополнительно имеет смысл вообще без провайдера протестировать. Как будет в большинстве случаев.

Reply:
Согласен. Добавлю везде вариант без провайдера